Formula/pyside.rb

```diff
@@ -61,18 +62,5 @@ def install
         Xml
       ].each { |mod| system python, "-c", "import PySide2.Qt#{mod}" }
     end
-    ["python", "python@2"].each do |python|
```

Comment:
Why is that part of the test removed?

Reply:
As I said before, importing the python bindings worked without `llvm` but, due to the opportunistic linkage, `llvm` was still required and a separate test was needed to check if this linkage was correct. So, I had to create a shiboken2-based test (see [here](https://github.com/Homebrew/homebrew-core/pull/29699/commits/0d607c220ba4cd3cd70e2a791d7a63aa87301734)).  Now the linkage does not require `llvm` anymore, so there is no need to embed shiboken in the `pyside` formula, hence this test is not required.